google-compute-engine-auto-updater.goospec

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "google-compute-engine-auto-updater",
+  "version": "1.0.0@2",
+  "arch": "noarch",
+  "authors": "Google Inc.",
+  "license": "http://www.apache.org/licenses/LICENSE-2.0",
+  "description": "Google Compute Engine auto updater for core packages",
```

Comment:
while its all packages right now we will release packages that dont get auto updated, maybe 'core Google packages' ?